foundry/pixi/containers/wallsLayer.d.ts

```diff
@@ -0,0 +1,155 @@
+import 'pixi.js';
```

Comment:
This file should go to `foundry/pixi/containers/canvasLayers/placeablesLayers/wallsLayer.d.ts`.